universal-application-tool-0.0.1/app/views/LoginForm.java

```diff
@@ -28,55 +35,134 @@ public LoginForm(BaseHtmlLayout layout, Config config) {
+        this.layout
+            .viewUtils
+            .makeLocalImageTag("ChiefSeattle_Blue")
+            .withClasses(Styles.W_1_4, Styles.PT_4));
+    content.with(
+        div()
+            .withClasses(Styles.FLEX, Styles.TEXT_4XL, Styles.GAP_1, Styles._MT_6, Styles.PX_8)
+            .with(p("Seattle").withClasses(Styles.FONT_BOLD))
+            .with(p("CiviForm")));
+
+    String loginMessage = "Please log in with your City of Seattle account";
+    content.with(
+        div()
+            .withClasses(
+                Styles.FLEX,
+                Styles.FLEX_COL,
+                Styles.GAP_2,
+                Styles.PY_6,
+                Styles.PX_8,
+                Styles.TEXT_LG,
+                Styles.W_FULL,
+                Styles.PLACE_ITEMS_CENTER)
+            .with(p(loginMessage))
+            .with(loginButton()));
+
+    String alternativeMessage = "Don't have an account?";
+    String or = "or";
+    content.with(p(alternativeMessage).withClasses(Styles.TEXT_LG));
+    ContainerTag alternativeLoginButtons =
+        div()
+            .withClasses(
+                Styles.PB_12,
+                Styles.PX_8,
+                Styles.FLEX,
+                Styles.GAP_4,
+                Styles.ITEMS_CENTER,
+                Styles.TEXT_LG);
+    if (config.hasPath("idcs.register_uri")) {
+      alternativeLoginButtons.with(createAccountButton()).with(p(or)).with(guestButton());
+    } else {
+      alternativeLoginButtons.with(guestButton());
     }
+    content.with(alternativeLoginButtons);
 
-    return layout.render(htmlBundle);
+    String somethingElse = "Looking for something else?";
+    content.with(
+        div()
+            .withClasses(
+                Styles.BG_GRAY_100,
+                Styles.PY_4,
+                Styles.PX_8,
+                Styles.W_FULL,
+                Styles.FLEX,
+                Styles.GAP_2,
+                Styles.JUSTIFY_CENTER,
+                Styles.ITEMS_CENTER,
+                Styles.TEXT_BASE)
+            .with(p(somethingElse).with(text(" ")).with(adminButton())));
+
+    return div()
+        .withClasses(Styles.FIXED, Styles.W_SCREEN, Styles.H_SCREEN, Styles.BG_GRAY_200)
+        .with(content);
+  }
+
+  private ContainerTag debugContent() {
+    return div()
+        .withClasses(Styles.ABSOLUTE)
+        .with(
+            h1("DEBUG MODE: BECOME ADMIN"),
+            redirectButton(
+                "admin",
+                "Global",
+                routes.CallbackController.fakeAdmin(
+                        FakeAdminClient.CLIENT_NAME, FakeAdminClient.GLOBAL_ADMIN)
+                    .url()),
+            redirectButton(
+                "program-admin",
+                "Of All Active Programs",
+                routes.CallbackController.fakeAdmin(
+                        FakeAdminClient.CLIENT_NAME, FakeAdminClient.PROGRAM_ADMIN)
+                    .url()));
+  }
+
+  private Tag loginButton() {
+    String msg = "LOG IN";
```

Comment:
suggestion (blocking): In general, these should be written in Sentence case ("Log in", "Create account", "Continue as guest"). There is style that transforms them to upper case (https://tailwindcss.com/docs/text-transform), which I think we already use in our base applicant styles.

Reply:
done!